scripts/bump.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs");
+const path = require("path");
+const { exec } = require("@actions/exec");
+
+(async () => {
+  await exec("pnpm changeset", ["version"]);
+  const apps = fs.readdirSync("apps");
```

Comment:
The only thing I _may_ add here is to filter directories only (in case there is `apps/README.md` or anything other flat file.